src/app/shared/helpers/config.ts

```diff
@@ -17,7 +22,7 @@ export type Environment = {
   defaultActivityId: string;
   defaultSkillId: string;
 
-  languages?: { tag: string, path: string }[];
+  languages: LanguageConfig[];
```

Comment:
There must be an explanation on how to use this (or/and an example), if the match all "/" has to be at the end, it must be mentioned.

Reply:
Fixed